tests/TestRunner/main.cpp

```diff
@@ -30,7 +30,12 @@ static constexpr const wchar_t* g_applications[] =
     L"CompositionTest_8wekyb3d8bbwe!Fixed",
     L"FileSystemTest_8wekyb3d8bbwe!Fixed",
     L"LongPathsTest_8wekyb3d8bbwe!Fixed",
-    L"WorkingDirectoryTest_8wekyb3d8bbwe!Fixed"
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTNoScripts",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTOnlyStart",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTOnlyEnd",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTBothStartingFirst",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTBothEndingFirst",
+    L"PowershellScriptTest_8wekyb3d8bbwe!PSTScriptWithArg"

```

Comment:
Do we care about tests running in and out of the container?

Reply:
Right.  I should add those.